src/gui.js

```diff
@@ -8240,10 +8251,7 @@ ProjectDialogMorph.prototype.unshareProject = function () {
                         entry.label.rerender();
                         this.buttons.fixLayout();
                         this.rerender();
-                        this.ide.showMessage('unshared.', 2);
-                        if (proj.projectname === ide.getProjectName()) {
-                            location.hash = '';
-                        }
+                        this.ide.showMessage('unshared.', 2
```

Comment:
```suggestion
                        this.ide.showMessage('unshared.', 2);
```

Ugh, this was a bad merge